builtin/methods.go

```diff
@@ -386,3 +386,9 @@ var MethodsEAM = struct {
 	2,
 	3,
 }
+
```

Comment:
This file needs embryo methods

Reply:
It doesn't have any methods (not even a constructor), but I'll add an empty list here.